src/uk/gov/hmcts/contino/GithubAPI.groovy

```diff
@@ -180,4 +180,34 @@ class GithubAPI {
   def checkForDependenciesLabel(branch_name) {
     return checkForLabel(branch_name, "dependencies")
   }
+
+  /**
+   * Calls workflow to manually startup environment
+   * @param workflowName
+   *   The file name of the workflow to run 'manual-start.yaml'
+   * @param businessArea
+   *   I.e CFT
+   * @param cluster
+   *   AKS Cluster number, i.e 00
+   * @param environment
+   *   Environment to start
+  */
+  def startAksEnvironmentWorkflow(String workflowName, String businessArea, String cluster, String environment){
+    def body = """{"ref":"master","inputs":{"PROJECT":"${businessArea}", 
+    "SELECTED_ENV":"${environment}","AKS-INSTANCES":"${cluster}"}}"""
```

Comment:
something like this would be easier to read:
```suggestion
    def body = """
      { 
        "ref":"master", 
        "inputs":{
           "PROJECT": "${businessArea}", 
           "SELECTED_ENV": "${environment}",
           "AKS-INSTANCES": "${cluster}"
         }
       }
    """
```